OpenDreamServer/Dream/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -79,6 +79,31 @@ static class DreamProcNativeRoot {
 
             return new DreamValue(Convert.ToChar(ascii).ToString());
         }
+        
+        [DreamProc("clamp")]
+        [DreamProcParameter("Value")]
```

Comment:
This parameter can be either a number, or a DreamList (a DreamObject)

```suggestion
        [DreamProcParameter("Value", Type = DreamValueType.Number | DreamValueType.DreamObject)]
```